mixing/mixclient/client.go

```diff
@@ -2245,8 +2280,8 @@ func (c *Client) alternateSession(ps *pairedSessions, prs []*wire.MsgMixPairReq)
 		}
 		pr := prsByHash[prHashByIdentity[ke.Identity]]
 		if pr == nil {
-			err := fmt.Errorf("missing PR %s by %x, but have their KE %s",
-				prHashByIdentity[ke.Identity], ke.Identity[:], ke.Hash())
+			err := fmt.Errorf("missing PR by %x, but have their KE %s",
+				ke.Identity[:], ke.Hash())
```

Comment:
had this fix sitting in my repo as well, i can break it out into another pr if it matters, just fixes up an invalid log (the pr hash logged here is always all zeros)

Reply:
I don't mind it being in this PR.  The `mixclient` is really only for wallet, so I'm not as pick on it.